tests/components/sharePanel-test.js

```diff
@@ -70,11 +69,79 @@ describe('SharePanel', function () {
         TestUtils.Simulate.click(twitter);
         TestUtils.Simulate.click(facebook);
         TestUtils.Simulate.click(googlePlus);
+        OO={};
+        OO.isIos = false;
+        OO.isSafari = false;
         TestUtils.Simulate.click(emailShare);
+	OO=null;
```

Comment:
the indentation here is wrong. And it needs to be OO = null;

Reply:
Fixed indentation and spacing issue.